docs/detections/rules-ui-monitor.asciidoc

```diff
@@ -15,6 +27,30 @@ TIP: To sort by any other column, switch on the *Technical preview* toggle above
 
 For detailed information on a rule, the alerts it generated, and associated errors, click on a rule's name in the *All rules* table. This also allows you to perform the same actions that are available on the <<rules-ui-management, **Rules** tab>>, such as modifying or deleting rules, activating or deactivating rules, exporting or importing rules, and duplicating prebuilt rules.
 
+[float]
+[[rule-execution-logs]]
+=== Rule execution logs
+
+Each detection rule keeps a log of every time it ran, its success or failure, any warning or error messages, and how long it took to search for data, create alerts, and complete its execution. This can help you troubleshoot a particular rule if it isn't behaving as expected (for example, if it isn't creating alerts or takes a long time to run).
```

Comment:
```suggestion
Each detection rule execution is logged, including its success or failure, any warning or error messages, and how long it took to search for data, create alerts, and complete. This can help you troubleshoot a particular rule if it isn't behaving as expected (for example, if it isn't creating alerts or takes a long time to run).
```
I could be wrong but my thinking here is that the rule itself does not keep a log, the platform does.